src/tools/manifest2proto.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {Runtime} from '../runtime/runtime.js';
+import {ErrorSeverity, Manifest, ManifestError} from '../runtime/manifest.js';
+import protobuf from 'protobufjs';
+
+const rootNamespace = protobuf.loadSync('./java/arcs/core/data/manifest.proto');
+const manifestProto = rootNamespace.lookupType('arcs.Manifest');
+
+export async function serialize2proto(path: string): Promise<Uint8Array> {
+  const manifest: Manifest = await Runtime.parseFile(path);
+
+  if (manifest.errors.some(e => e.severity === ErrorSeverity.Error)) {
+    throw new Error(`Problems found in manifest '${path}'`);
```

Comment:
Can you include the errors in the error message?

Reply:
This whole thing is actually not needed, Runtime.parseFile will throw for errors. I've removed this.